### ISSUE TITLE
Signs in Images Blurred out

### ISSUE THREAD

**Issue by MikeTho16** [openstreetview/openstreetview.org#27]
See the area circled in red in the image below:
![image](https://cloud.githubusercontent.com/assets/15094680/17222439/7a7f8d72-54b5-11e6-9acb-37963abfad9f.png)

Although we can still see that it is a stop sign, it would be nice to be able to read the name of the road on the sign above it.

**Comment by martijnv-telenav** [comment]
This is a hard problem and we are continuously working on improving the detection algorithms. Sorry I can't give you a more specific answer. I will leave this open so others can report their cases as well.

**Comment by MikeTho16** [comment]
@martijnv-telenav yes, this is a difficult problem.  A lot of things look like a face or a license plate, which I presume are the two primary things you are trying to blur.  Perhaps you could maintain a copy of the original image on your servers and allow the user to selectively "un-blur" parts of an image.

**Comment by Zverik** [comment]
This is definitely important. Almost all signs on my photos are blurred. E.g. http://openstreetview.org/details/10984/86 and http://openstreetview.org/details/10984/73

**Comment by drkludge** [comment]
I used horatiu's images along I-40 to add a speed limit sign.  The whole series of images has false positives on signs and parts of the desert along the way.  The most problematic blurs are the two images on a standard green US information sign.  The first image is completely useless after the blurs.  The second image escapes the blurs but by then the image shows the sign mostly cut off.  The kicker is the image of the semi trailer.  There is not a single blur. Just as with Mapillary, OSV is less than it could be with all this blurring that goes on.  If I want to use Mapillary as it is, then I have saturate an area with images so that I can find enough detail about all four numbers in an address.  If this is too hard to do, then please don't blur the images.  
http://openstreetview.com/details/7036/84
http://openstreetview.com/details/7036/85
http://openstreetview.com/details/7036/131
http://www.openstreetmap.org/node/4336601813

**Comment by Jakkajakka** [comment]
http://www.openstreetview.org/details/1159/1951
Not only small sign on poles are detected.
Information Signs. Big blue bords with white text, direction indications are also blurred. These are higher positioned then license plate. Succes

**Comment by Hjart** [comment]
I tried using OpenStreetCam images for mapping today, but blurs like in https://openstreetcam.org/details/9294/786 makes them practically useless. At least with Mapillary I can request that such images are unblurred.

**Comment by ithink314** [comment]
Saw several like this, and it makes me question the value/importance of intentionally blurring anything if obscured signs is the result. IANAL, but this does make one wonder about expectations when traveling on public streets.

https://storage9.openstreetcam.org/files/photo/2018/12/14/proc/1314347_6c258_1481.jpg

**Comment by daviewales** [comment]
Even just the option to unblur our own imagery would be good.
I always go through my own uploads to delete any bad / overexposed images, etc.
If there was a button I could click to unblur false positives, or a way we could help with the training, like we can for street signs, that would be great.

**Comment by mirunac-telenav** [comment]
A new blurring mechanism is currently under development.
Will come back to this thread to update everyone when it will be deployed. 
Sorry for taking this long to receive a response. 
We're going to allocate more resources to Git as of this year, too. 
Tx

**Comment by JokerGermany** [comment]
>At least with Mapillary I can request that such images are unblurred.
Yeah, no possiblity to report such images is really sad.

Another Example:
https://openstreetcam.org/details/1336551/974/track-info

**Comment by daviewales** [comment]
Here is some blurring inside a sign which has a valid detection!

https://openstreetcam.org/details/1343611/609/detections

**Comment by JokerGermany** [comment]
https://openstreetcam.org/details/1466774/911/track-info
